pandas/io/stata.py

```diff
@@ -162,13 +162,13 @@
 --------
 Read a Stata dta file:
 
->>> df = pd.read_stata('filename.dta')
+>>> df = pd.read_stata('docs_example_files/read_stata_example.dta')
```

Comment:
I've created this directory to store this dummy Stata file in pandas/core/io. ie the same base directory as this file but feel free to tell me to move the directory to a more appropriate place.

Reply:
You could construct a DataFrame, and do a `to_stata` in the doc section first. Then read it. Some of the other IO functions are like that.

See https://github.com/pandas-dev/pandas/blob/c86b129e517ff5489160d69b8eb399b68488b82e/pandas/io/pickle.py#L69